src/shared/util.js

```diff
@@ -45,7 +45,8 @@ var ImageKind = {
 var AnnotationType = {
   WIDGET: 1,
   TEXT: 2,
-  LINK: 3
+  LINK: 3,
```

Comment:
Thanks to PR #6757 the changes in this file are no longer necessary as types `MOVIE` and `SCREEN` are now already available when you rebase this patch.